core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -100,6 +117,8 @@ class ActivationThrottler(consulServer: String, concurrencyLimit: Int)(
                     val loadbalancerCount = loadbalancerActivationCount(subject)
                     subject -> (loadbalancerCount - invokerCount)
             }
+            publishUserConcurrentActivation()
```

Comment:
Given that above method returns a `List[Future[Unit]]` this will not "block" the scheduler long enough and could potentially lead to overlapping calls.

Can we change `publishUserConcurrentActivation` to return a `Future[Unit]`?